### PR TITLE
Use published packages and remove duplicated code

### PR DESCRIPTION
**Pull request by tylerbutler**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,22 +13,24 @@
...
     },
     "dependencies": {
-        "@fluid-experimental/data-objects": "^0.34.0-14942",
-        "@fluidframework/aqueduct": "^0.32.1",
-        "@fluidframework/get-tinylicious-container": "^0.32.1",
-        "@fluidframework/map": "^0.32.1",
+        "@fluid-experimental/data-objects": "^0.35.2",
+        "@fluid-experimental/experimental-fluidframework": "^0.35.2",
+        "@fluid-experimental/fluid-static": "^0.35.2",
+        "@fluidframework/aqueduct": "^0.35.2",
```
Comment (tylerbutler) [inline package.json:19]: Could remove this...

File: package.json [hunk 2]
```diff
@@ -13,22 +13,24 @@
...
+        "@fluidframework/get-tinylicious-container": "^0.34.1",
```
Comment (tylerbutler) [inline package.json:20]: Do I need this?

File: package.json [hunk 3]
```diff
@@ -13,22 +13,24 @@
...
+        "@fluidframework/map": "^0.35.2",
```
Comment (tylerbutler) [inline package.json:21]: Could remove this...

File: src/app.ts [hunk 4]
```diff
@@ -16,15 +17,19 @@ async function start(): Promise<void> {
     let keyValueDataObject: IKeyValueDataObject;
 
     if (isNew) {
-        const fluidDocument = await Fluid.createContainer(containerId, [
-            KeyValueInstantiationFactory,
-        ]);
+        const fluidDocument = await Fluid.createDocument(
+            containerId,
```
Comment (tylerbutler) [inline src/app.ts:21]: Feels a little odd to pass a container ID to createDocument...

File: package.json (hunk 1 above)
Comment (micahgodbolt) [reply package.json:19]: yes, we should only need data-objects and fluid-static. even the `experimental-fluidframework` can go

File: package.json (hunk 2 above)
Comment (micahgodbolt) [reply package.json:20]: nope

File: src/app.ts (hunk 4 above)
Comment (micahgodbolt) [reply src/app.ts:21]: i think we changed the name to documentId in other examples

File: package.json [hunk 5]
```diff
@@ -13,22 +13,20 @@
...
-        "@types/react": "^17.0.0",
-        "@types/react-dom": "^17.0.0",
+        "@types/react": "^17.0.2",
+        "@types/react-dom": "^17.0.1",
         "clean-webpack-plugin": "^3.0.0",
         "concurrently": "^5.3.0",
         "html-webpack-plugin": "^4.3.0",
-        "tinylicious": "0.4.11798",
-        "ts-loader": "^8.0.12",
+        "tinylicious": "0.4.17169",
```
Comment (micahgodbolt) [inline package.json:28]: you can also add `npx tinylicious` to the readme steps and leave it out as a dependency

**Review by micahgodbolt — APPROVED** [review]
Looks good!

**Pull request merged**